Minecraft Central/dist/metadata.json

```diff
@@ -0,0 +1,24 @@
+{
+    "author": {
+      "name": "hi_feeeh",
+      "id": "160923526277103617"
+    },
+    "service": "Minecraft Central",
+    "description": {
+      "en": "Play Minigames, Factions, Skyblock, Prison, Survival and much more on Minecraft Central!"
+    },
+    "url": [
+        "mccentral.org",
+        "buy.mccentral.org"
+      ],
+    "version": "1.0.0",
+    "logo": "https://i.imgur.com/W7XIUCd.png",
+    "thumbnail": "https://i.imgur.com/yTWQ5RJ.png",
+    "color": "#da8a00",
+    "tags": [
+        "minecraft",
+        "mccentral"
+      ],
+    "category": "games",
+    "iframe": false

```

Comment:
can remove the iframe line, dont forget to remove the comma behind games if you do

Reply:
Done.